### DISCUSSION TITLE
can you add some realistic diffusion like dream like etc

### DISCUSSION THREAD

**Discussion by Hareshkumar77** [etherealxx/etherportal-webui-colab#1] [Ideas]
add some diffusion based on photorealistic

**Comment by etherealxx** [comment]
I appreciate your request.

However, i want the UI to not be confusing and cluttered. If i want, i could, include every colab on camenduru's main, like [the thing that this dude trying to achieve](https://colab.research.google.com/github/camenduru/stable-diffusion-webui-colab/blob/community/machiavel23/Global_webui_colab.ipynb).

I don't want to confuse the beginners with many models at once. That's why i made [this comparison page](https://github.com/etherealxx/etherportal-webui-colab/blob/master/ModelComparison.md). Every model there is curated, based on their ability to make clear image by just `masterpiece, best quality, (girl name)`. Again, to make people that are familiar with NovelAI can learn quickly. (Though, that page haven't been updated for months, i was busy). I also might remove models that aren't popular again (not as good as the others).

For now, i'll focus on the anime models first, fixing some bugs, and adding more features. And then i'll reconsider your suggestion.

Thank you.
